packages/esm-patient-conditions-app/src/conditions/conditions-form.scss

```diff
@@ -3,16 +3,17 @@
 @import "~carbon-components/src/globals/scss/mixins";
 
 .condition {
-  padding: 0.625rem;
+  padding: 0.875rem;
 }
 
 .conditionsList {
   background-color: $ui-02;
+  max-height: 14rem;
+  overflow-y: scroll;
```

Comment:
AFAIK this will *always* show the scrollbar. Is this intentional? Should it rather be `auto`?

Reply:
No, it's not. Changed to `auto`.